### ISSUE TITLE
Too much disk clutter

### ISSUE THREAD

**Issue by DevelopCMD** [GanerCodes/videoEditBot#69]
So whenever VideoEditBot receives a command, it generates the video and puts it in the active directory. But the thing is, it doesn't delete it, which means that too many of these files will take up too much disk space. Is there a way for the bot to safely delete the generated video when it is done uploading/processing it? I am running a self-host of this bot, and it would be useful if this feature was added.

**Comment by sinogrime** [comment]
It does delete it - I'm not sure what's wrong with your version of the bot but every time a file is processed by the bot it's deleted for me.

**Comment by GanerCodes** [comment]
yeah `keepExtraFiles=True` inside the discordBot.py, so make sure your up to date and whatnot. It works fine for me as well

**Comment by HBDeveloper11** [comment]
thanks, i'll enable this feature soon, it really helps (btw this is my alt account, my main account is DevelopCMD)

**Comment by JojjeE** [comment]
I've just launched my own instance of VEB pulling the latest repo and it seems to keep uploaded videos in the active directory. I looked in discordBot.py and the line is `Action(editor, swap_arg("filename"), args, workingDir = working_directory, keepExtraFiles = False,`
Only been running for an hour or two, maybe it cleans it up after a set time or something? I'll keep it running and see what it does.

Or do you mean I should set it to true? It's kind of a misnomer if that's so...
